shared/interfaces/block_interface.go

```diff
@@ -43,6 +44,7 @@ type BeaconBlockBody interface {
 	Attestations() []*ethpb.Attestation
 	Deposits() []*ethpb.Deposit
 	VoluntaryExits() []*ethpb.SignedVoluntaryExit
+	SyncAggregate() *ethpb.SyncAggregate
```

Comment:
The Beacon Block Body Should be able to be shared by all beacon blocks not just altair blocks. It seems unsafe to allow
phase 0 blocks to have this method.